OpenRA.Mods.Common/Traits/GivesBounty.cs

```diff
@@ -58,6 +58,29 @@ int GetMultiplier(Actor self)
 			return (slevel > 0) ? slevel * info.LevelMod : 100;
 		}
 
+		int GetBountyValue(Actor self, int percentage)
```

Comment:
Passing the percentage in here is redundant.  You only ever call this with `info.Percentage`.